app/v1/controllers/requestParsers/validators/validations/package.scala

```diff
@@ -16,8 +16,15 @@
 
 package v1.controllers.requestParsers.validators
 
+import java.time.LocalDate
```

Comment:
It would be better if we replace package objects either with **top-level definitions**. If the team is fine with package objects I have no issues